ape_trezor/client.py

```diff
@@ -32,27 +33,40 @@ def __init__(self, hd_root_path: HDBasePath):
         self._hd_root_path = hd_root_path
 
     def get_account_path(self, account_id: int) -> str:
+        account_path = str(self._hd_root_path.get_account_path(account_id))
         try:
-            return ethereum.get_address(
-                self._client, parse_hdpath(f"{self._hd_root_path}/{account_id}")
-            )
+            return ethereum.get_address(self._client, parse_hdpath(account_path))
         except (PinException, TrezorFailure) as exc:
             message = "You have entered an invalid PIN."
             raise TrezorAccountException(message) from exc
 
 
-def _to_vrs(reply: bytes) -> Tuple[int, bytes, bytes]:
+class TrezorSignature(BaseModel):
     """
-    Breaks a byte message into 3 chunks vrs,
-    where `v` is 1 byte, `r` is 32 bytes, and `s` is 32 bytes.
+    Class representing a Trezor Signature
     """
-    if not reply:
-        raise TrezorClientError("No data in reply.")
 
-    v = reply[0]  # 1 byte
-    r = reply[1:33]  # 32 bytes
-    s = reply[33:65]  # 32 bytes
-    return v, r, s
+    v: int
+    r: bytes
+    s: bytes
+
+    def __init__(self, **data: Any) -> None:
+        """
+        Pass `v`: int, `r`: bytes, and  `s`: bytes fields directly
+        or an array of bytes as `signature_bytes` to be broken into 3 chunks
+        vrs, where `v` is 1 byte, `r` is 32 bytes, and `s` is 32 bytes.
+        """
+        if "signature_bytes" in data:
+            signature_bytes = data.pop("signature_bytes")
+
+            if signature_bytes is None:
+                raise TrezorClientError("No data in signature bytes.")
+
+            data["v"] = signature_bytes[0]  # 1 byte
+            data["r"] = signature_bytes[1:33]  # 32 bytes
+            data["s"] = signature_bytes[33:65]  # 32 bytes
+
+        super().__init__(**data)
```

Comment:
I don't think you need a whole class and to introduce pydantic to do all of this, just do something like this instead:

```py
from ape.types import MessageSignature, TransactionSignature

def extract_signature_vrs_bytes(data) -> (int, bytes, bytes):
    if "signature_bytes" not in data:
        raise Exception()

    signature_bytes = data.pop("signature_bytes")
    if signature_bytes is None:
        raise TrezorClientError("No data in signature bytes.")

    return signature_bytes[0], signature_bytes[1:33], signature_bytes[33:65]

...

sig = MessageSignature(*extract_signature_vrs_bytes(tuple_reply))

...

sig = TransactionSignature(*extract_signature_vrs_bytes(tuple_reply))
```